src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -141,25 +141,17 @@ export default {
 	&__date-header {
 		display: block;
 		text-align: center;
-
-		margin: 40px 15px 0;
-		border-top: 1px solid var(--color-border);
 		padding-top: 20px;
 		position: relative;
+		margin: 20px 0;
 
 		.date {
 			content: attr(data-date);
-			position: absolute;
-			top: 0;
-			left: 50%;
-			transform: translateX(-50%) translateY(-50%);
-			padding: 0 7px 0 7px;
-
-			text-align: center;
-			white-space: nowrap;
-
+			background-color: var(--color-background-dark);
+			font-weight: bold;
+			padding: 6px 12px;
```

Comment:
Also I would reduce the padding and make it is small as possible (similar to the mention chips).

Reply:
Mentions don't have sufficient padding for their font size. If I'm to reduce the padding I'd have to reduce the font size too, which I'm not too fond of...